icaruscode/TPC/Calorimetry/calorimetryICARUS.fcl

```diff
@@ -5,11 +5,13 @@ BEGIN_PROLOG
 icarus_calorimetryalgdata:	@local::standard_calorimetryalgdata
 icarus_calorimetryalgmc:	  @local::standard_calorimetryalgmc
 
-# Calorimetry constants from G. Putnam (Dec 2021)
-# Informed by truth based study on ICARUS MC -- docdb 23856
+# Calorimetry constants from G. Putnam (Mar 2022)
+# Informed by truth based study on ICARUS MC
+# docdb 23856 discusses the methodology for a different iteration of 
```

Comment:
Minor comment - Can you put the plots you threw in Slack to determine these constants in a doc-db with maybe a few bullet points motivating the change and the steps taken, just for posterity's sake? (and then update the comment here in the near future). Doesn't hold up me approving this PR but it would be good to have that info stored in doc-db.